### PR TITLE
eap8 resource added

### PR DESCRIPTION
**Pull request by YogiSoni-lazy**

### CODE REVIEW

File: eap8/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# -- mode: ruby --
+# vi: set ft=ruby :
+# Load variables from the config file #
+config_file = File.join(File.dirname(FILE), "variables.yml")
+vars = YAML.load_file(config_file)
+Vagrant.configure("2") do |config|
+  # Global Configuration  #
+  #config.vm.box = "generic-x64/rhel9"
+  config.vm.box_url = "http://content.syslab.pnq2.redhat.com/content/aap/rhel9/metadata.json"
```
Comment (rsriniva) [inline eap8/Vagrantfile]: This URL looks like it is inside Red Hat VPN - Copy the JSON file to this repo.

Comment (rsriniva) [reply eap8/Vagrantfile]: Remember, partners cannot access Red Hat VPN resoruces

File: eap8/Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  config.vm.box = "nshaik/rhel9"
```
Comment (rsriniva) [inline eap8/Vagrantfile]: Where is the box file coming from? I think we need to leave instructions to download the raw RHEL 9 ISO and then covert it to a vagrant box file. We can also replace with CentOS Stream 9

File: eap8/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+= You can optionally get started using vagrant along with libvirt that would configure below settings.
```
Comment (rsriniva) [inline eap8/README.md]: Rename this eap8 folder to something like eap8-vagrant and move it to the admin1 folder at the top-level of this repo

**Review by rsriniva — CHANGES_REQUESTED** [review]

File: eap8/Vagrantfile (hunk 1 above)
Comment (YogiSoni-lazy) [reply eap8/Vagrantfile]: Updated the box details

File: eap8/Vagrantfile (hunk 2 above)
Comment (YogiSoni-lazy) [reply eap8/Vagrantfile]: generic-x64/rhel9 comes from vagrant Hub

File: admin1/eap8-vagrant/Vagrantfile [hunk 4]
```diff
@@ -5,9 +5,7 @@ config_file = File.join(File.dirname(FILE), "variables.yml")
 vars = YAML.load_file(config_file)
 Vagrant.configure("2") do |config|
   # Global Configuration  #
-  #config.vm.box = "generic-x64/rhel9"
-  config.vm.box_url = "http://content.syslab.pnq2.redhat.com/content/aap/rhel9/metadata.json"
-  config.vm.box = "nshaik/rhel9"
+  config.vm.box = "generic-x64/rhel9"
```
Comment (rsriniva) [inline admin1/eap8-vagrant/Vagrantfile:8]: is this box available on vagrant vault repo?

**Review by rsriniva — APPROVED** [review]
Approved

**Pull request merged**